clients/acm.js

```diff
@@ -0,0 +1,14 @@
+require('../lib/node_loader');
+var AWS = require('../lib/core');
+var Service = require('../lib/service');
```

Comment:
`core.js` already requires `service.js`, and the `Service` constructor can already be accessed at `AWS.Service`. Why does it need to be required here? We also wouldn't need to export `Service` in `service.js`